docs/source/conf.py

```diff
@@ -55,6 +55,7 @@
     "sphinx.ext.napoleon",
     "sphinx.ext.viewcode",
     "sphinx.ext.githubpages",
+    "sphinxcontrib.jquery",
```

Comment:
As you mentioned in https://github.com/optuna/optuna/pull/4615#issuecomment-1510024686, this fix is a workaround for https://github.com/readthedocs/sphinx_rtd_theme/issues/1452. How about adding a TODO comment to remove it after resolving the issue?

Reply:
Thank you for your indication. I added a TODO comment.